training/model_management/components/download_model/spec.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+
+name: download_model
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Download model
+description: Downloads a model from a public storage container
```

Comment:
> storage container [](http://example.com/codeflow?start=44&length=18)

Could we add more description here that talks about various sources and examples?

Reply:
thanks Sasi. Adding in the next PR in favour of closing this one.